lib/emittr/events.rb

```diff
@@ -1,3 +1,5 @@
+require_relative 'callback'
```

Comment:
Shouldn't we add this here? https://github.com/talyssonoc/emittr/blob/master/lib/emittr.rb#L3